LICENSE.md

```diff
@@ -67,58 +67,59 @@ The User may not limit access to the Mod in any way, except as provided by the G
 The User may decompile compiled binaries of the Mod.  
 The User may examine the Mod's source code.
 
-5. Right to distribute
+5. No right to distribute
 --------
 
-The Author reserves the right to distribute the Mod.  
-The Author may designate a person or organization as a Distributor.  
-A Distributor is granted the right to distribute the Mod.  
-The Author may require that a Distributor provides a direct link to the Codebase.  
-The User may not distribute the Mod without being granted distribution rights from the Author.  
-The Author may revoke distribution rights from any Distributor.  
+The Author reserves the right to transmit verbatim copies of ("Distribute") the Mod.  
+Rights to distribute the Mod must be granted in a separate license which must be explicitly obtained from the Author.
```

Comment:
Interesting. Their terms say they exercise no control over your content. Meaning the author has full control over the content. That's probably how they get around being considered a distributor. If they're not distributing your content, you don't have to license it to them. You're just using them as a tool to distribute your content yourself. I wonder why Curse doesn't do it that way.